sentences/en/light_HassLightSet.yaml

```
@@ -83,8 +83,8 @@ intents:
         response: "color"
 
       - sentences:
-          - "[<set>] [[the] color of] [all] <light> [to] {color}"
-          - "[<set>] [[the] color of] [all] <light> (<in_here>;[to] {color})"
+          - "[<set>] [[the] color of] [all] [<the>] <light> [to] {color}"
```

You could use `<all>` instead of `all`
```suggestion
          - "[<set>] [[the] color of] [<all>] <light> [to] {color}"
```

---

Hmm. Lots of other refactors here too. I'll try to make some minimal adjustments and then do a bigger refactor of the sentences as a whole in another PR.